src/plugins/management/public/management_sections_service.test.ts

```diff
@@ -105,4 +106,29 @@ describe('ManagementService', () => {
       ]
     `);
   });
+
+  it('should disable apps register in opensearchDashboards section', () => {
+    const originalDataSourcesCapability =
+      DEFAULT_MANAGEMENT_CAPABILITIES.management.opensearchDashboards.dataSources;
```

Comment:
nit: ideally we could mock this over setting it globally, but you do clean it up so not blocking.